### PR TITLE
[tensorflow] [build,test] Fix python sdk repo localmode tfs tests failing issue

### PR DESCRIPTION
**Pull request by schenqian**
*Issue #, if available:*
#985 
## PR Checklist
- [ ] I've prepended PR tag with frameworks/job this applies to : [mxnet, tensorflow, pytorch] | [ei/neuron] | [build] | [test] | [benchmark] | [ec2, ecs, eks, sagemaker]
- [ ] (If applicable) I've documented below the DLC image/dockerfile this relates to
- [ ] (If applicable) I've documented below the tests I've run on the DLC image
- [ ] (If applicable) I've reviewed the licenses of updated and new binaries and their dependencies to make sure all licenses are on the Apache Software Foundation Third Party License Policy Category A or Category B license list.  See [https://www.apache.org/legal/resolved.html](https://www.apache.org/legal/resolved.html).
- [ ] (If applicable) I've scanned the updated and new binaries to make sure they do not have vulnerabilities associated with them.

## Pytest Marker Checklist
- [ ] (If applicable) I have added the marker `@pytest.mark.model("<model-type>")` to the new tests which I have added, to specify the Deep Learning model that is used in the test (use `"N/A"` if the test doesn't use a model)
- [ ] (If applicable) I have added the marker `@pytest.mark.integration("<feature-being-tested>")` to the new tests which I have added, to specify the feature that will be tested
- [ ] (If applicable) I have added the marker `@pytest.mark.multinode(<integer-num-nodes>)` to the new tests which I have added, to specify the number of nodes used on a multi-node test
- [ ] (If applicable) I have added the marker `@pytest.mark.processor(<"cpu"/"gpu"/"eia"/"neuron">)` to the new tests which I have added, if a test is specifically applicable to only one processor type

## Reviewer Checklist
* For reviewer, before merging, please cross-check:
- [ ] I've verified the code change on the config file mentioned above has already been reverted

*Description:*
Sync https://github.com/aws/sagemaker-tensorflow-serving-container/pull/194

*Tests run:*

*DLC image/dockerfile:*

*Additional context:*

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license. I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by saimidu** [comment]
@schenqian Please explain what the issue is in more detail, or place a reference to the GitHub Issue/PR relevant to the issue with a short explanation, in the `Description` section of the PR.

Also, since this PR addresses a test failure, and we see no SageMaker TFS tests failing in the DLC GitHub repo, please add the relevant test to the repo that will allow us to validate that this issue has been fixed.

**Comment by schenqian** [comment]
@Sai, this commit is to address localmode tfs tests failure in sagemaker python sdk repo, related build failure details are here: https://paste.amazon.com/show/chqa/1616448415 
[36my8bc30vt0p-algo-1-rqncd |[0m Traceback (most recent call last):
[36my8bc30vt0p-algo-1-rqncd |[0m   File "/sagemaker/serve.py", line 383, in <module>
[36my8bc30vt0p-algo-1-rqncd |[0m     ServiceManager().start()
[36my8bc30vt0p-algo-1-rqncd |[0m   File "/sagemaker/serve.py", line 370, in start
[36my8bc30vt0p-algo-1-rqncd |[0m     self._setup_gunicorn()
[36my8bc30vt0p-algo-1-rqncd |[0m   File "/sagemaker/serve.py", line 164, in _setup_gunicorn
[36my8bc30vt0p-algo-1-rqncd |[0m     self._tfs_grpc_port_range, self._tfs_rest_port_range,
[36my8bc30vt0p-algo-1-rqncd |[0m AttributeError: 'ServiceManager' object has no attribute '_tfs_grpc_port_range'

**Review by liangma8712 — APPROVED** [review]

**Comment by tejaschumbalkar** [comment]
@schenqian Can you add relevant test on the DLC repo that verifies the fix as recommended by @saimidu?

File: src/config/build_config.py [hunk 1]
```diff
@@ -5,9 +5,9 @@
 # Do remember to revert it back to False before merging any PR (including NEURON dedicated PR)
 ENABLE_NEURON_MODE = False
 # Frameworks for which you want to disable both builds and tests
-DISABLE_FRAMEWORK_TESTS = []
+DISABLE_FRAMEWORK_TESTS = ["mxnet", "pytorch"]
```
Comment (tejaschumbalkar) [inline src/config/build_config.py:8]: Please revert this change for PR to be merge ready

**Review by tejaschumbalkar — APPROVED** [review]
